modules/standard/BigInteger.chpl

```diff
@@ -2595,6 +2628,24 @@ When ``numer/denom`` does not produce an integer, this method may produce incorr
       return false;
   }
 
+  /*
+    Return ``true`` if ``this`` is congruent to ``con`` modulo ``mod``.
+    ``this`` is congruent to ``con % mod`` if there exists an integer ``q``
+    satisfying ``this = con + q*mod``.  Unlike the other division functions,
+    ``d = 0`` is accepted.  As a result ``this`` and ``con`` are considered
```

Comment:
Same as above, this can be confusing, but I don't have a problem with keeping it like this